.github/workflows/code-quality.yml

```
@@ -0,0 +1,279 @@
+
+      - name: 📊 Bundle analysis
+        run: |
+          # Install bundle analysis tools
+          npm install -g bundlesize
+          
+          # Analyze bundle sizes
+          echo "Analyzing bundle sizes..."
+          ls -la dist/ || ls -la build/ || echo "No build directory found"
+
+      - name: 🚀 Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v10
```

actionlint (1.7.7)

the runner of "treosh/lighthouse-ci-action@v10" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)